packages/destination-actions/src/destinations/google-enhanced-conversions/uploadConversionAdjustment/index.ts

```diff
@@ -225,7 +228,8 @@ const action: ActionDefinition<Settings, Payload> = {
       userAgent: payload.user_agent
     }
 
-    if (payload.adjustment_type == 'RESTATEMENT') {
+    // Google will return an error if this value is sent with retractions.
+    if (payload.adjustment_type != 'RETRACTION') {
```

Comment:
I would recommend always using strict equality:

```suggestion
    if (payload.adjustment_type !== 'RETRACTION') {
```